src/index.html

```diff
@@ -1,14 +1,32 @@
-<!doctype html>
+<!DOCTYPE html>
 <html lang="en">
   <head>
     <meta charset="UTF-8">
-    <meta name="viewport"
-          content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0">
-    <meta http-equiv="X-UA-Compatible" content="ie=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
     <title>Moyo header</title>
     <link rel="stylesheet" href="./style.css">
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link rel="preconnect" href="https://fonts.gstatic.com">
+    <link href="https://fonts.googleapis.com/css2?family=Roboto:wght@100&display=swap" rel="stylesheet">
   </head>
   <body>
-    <h1>Moyo header</h1>
+    <header class="header">
+      <a href="#" class="logo">
+        <img src="moyo.jpg" alt="Logo">
+      </a>
+      <nav>
+        <ul class="box">
+          <li class="blue1"><a href="#">APPLE</a></li>
+          <li class="blue2"><a href="#">SAMSUNG</a></li>
+          <li class="blue3"><a href="#">SMARTPHONES</a></li>
+          <li class="blue4 is-active" data-qa="hover"><a href="#">LAPTOPS & COMPUTERS</a></li>
```

Comment:
```suggestion
          <li class="nav__item" data-qa="hover"><a href="#" class="nav__link is-active">LAPTOPS & COMPUTERS</a></li>
```
is-active class should have link

Reply:
Still not fixed